pypeit/scripts/ql_vlt_fors2.py

```diff
@@ -0,0 +1,465 @@
+"""
```

Comment:
What does this method do that `QLMOS` does not?

I don't understand having these instrument specific QL methods..

Reply:
Right instrument specific is going to be phased out. I think we agree on that. The QL masters are going to be generated from afternoon calibrations as we have been discussing with DEIMOS.